linalg/densemat.cpp

```diff
@@ -4326,15 +4280,10 @@ DenseMatrixEigensystem::DenseMatrixEigensystem(DenseMatrix &m)
    ev.SetDataAndSize(NULL, n);
 
 #ifdef MFEM_USE_LAPACK
+   int layout = LAPACK_COL_MAJOR;
```

Comment:
`layout` could be a member of `DenseMatrixEigensystem`, depending on maintainer preferences.